examples_tests/common/CommonAPI.h

```diff
@@ -1,29 +1,97 @@
 #define _NBL_STATIC_LIB_
 #include <nabla.h>
-#include "nbl/system/CSystemWin32.h"
 #if defined(_NBL_PLATFORM_WINDOWS_)
-#include "nbl/system/CColoredStdoutLoggerWin32.h"
-#include <nbl/ui/CWindowWin32.h>
+#include <nbl/ui/CWindowManagerWin32.h>
+#include <nbl/system/ISystem.h>
+#include <nbl/system/CStdoutLogger.h>
+#include <nbl/system/CFileLogger.h>
+#include <nbl/system/CColoredStdoutLoggerWin32.h>
```

Comment:
they should all be in the relevant `nbl/namespace/declarations.h` and `nbl/namespace/definitions.h` which are included by `nabla.h.in`